frontend/__tests__/src/hooks/useExportCsvEffect.test.tsx

```diff
@@ -36,4 +37,17 @@ describe('use export csv effect', () => {
 
     expect(result.current.errorMessage).toEqual('failed to export csv: error message')
   })
+
+  it('should set error message when export csv response status 404', async () => {
+    csvClient.exportCSVData = jest.fn().mockImplementation(() => {
+      throw new NotFoundException('error message')
+    })
+    const { result } = renderHook(() => useExportCsvEffect())
+
+    act(() => {
+      result.current.fetchExportData(MOCK_EXPORT_CSV_REQUEST_PARAMS)
+    })
+    expect(result.current.isExpired).toEqual(true)
```

Comment:
give a `new line` follow our pattern

PLZ re-read our onboarding Mural and other docs

Reply:
done